x-pack/plugins/cases/public/components/add_comment/schema.tsx

```diff
@@ -22,7 +21,7 @@ export const schema: FormSchema<AddCommentFormSchema> = {
     type: FIELD_TYPES.TEXTAREA,
     validations: [
       {
-        validator: emptyField(i18n.COMMENT_REQUIRED),
+        validator: emptyField(''),
```

Comment:
Do we still need this schema file since we're not showing any errors anymore?

Reply:
We don't display the error but I thought some frontend validation wouldn't harm. 😅

Reply:
The `emptyField` validator will throw an error if the comment contains only empty characters. The current behavior is to show an error message to the user in this scenario. I think we should keep showing the error and change the error message to something more descriptive like "Empty comment is not allowed". Another alternative is to disable the button if the comment contains only empty characters but I found it a bit confusing as a user that I cannot press the button even though I typed something. The error message will inform me of the requirements.

Reply:
Yeah I agree with @cnasikas. I think what Michael was getting at in the issue was to not show an error if the user hasn't typed anything yet. So if there's an easy way to differentiate between _hasn't typed_ vs _typed all spaces_ we should show an error if it's all spaces but just disable the button if the user hasn't typed.